internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -110,13 +117,13 @@ func TestBasicAndOLM(t *testing.T) {
 			if err != nil {
 				t.Errorf("Invalid filepath")
 			}
```

Comment:
It shouldn't be necessary to get the absolute path. The working directory is the directory the test file is in.

Reply:
good catch, updated to not use abs paths.